exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return number > 0 && number < 10;
+}
```

Comment:
I understood this question differently: if number is between some numbers it means the function would include them, ie. >= and <=.

Reply:
I think you are right. I will change that one. Thank you.